library/oneview_server_profile.py

```diff
@@ -117,21 +117,21 @@
 RETURN = '''
 server_profile:
     description: Has the OneView facts about the Server Profile.
-    returned: On states 'present' and'compliant'
+    returned: On states 'present' and' compliant'
```

Comment:
shouldn't the space be after "and" and not before "compliant"?
See lines below as well